src/constants/ui.ts

```diff
@@ -2,3 +2,5 @@ import { ColorMode } from '@/types';
 
 export const DEFAULT_COLOR_MODE: ColorMode = 'light';
 export const COLOR_MODE_STORAGE_KEY = 'colorMode';
+
+export const HEADER_SCROLL_THRESHOLD = 250;
```

Comment:
How was this determined?

Reply:
It was hardcoded before in header. There is no particular reason for this value. It only determines how late header will be sticky to top after scoolling.